### PR TITLE
Closes #439: Write code that will estimate the match strength of every AffOrgMatchVoter

### LINKED ISSUE #439: Write code that will estimate the match strength of every AffOrgMatchVoter

### PR DESCRIPTION
**Pull request by lukdumi**

### CODE REVIEW

File: iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffOrgMatcherFactory.java [hunk 1]
```diff
@@ -84,17 +87,26 @@ public static AffOrgMatcher createDocOrgRelationMatcher(JavaSparkContext sparkCo
...
         // matcher
         
         return createAffOrgMatcher(docOrgRelationJoiner, docOrgRelationMatchComputer);
     }
+
+    /**
+     * Creates {@link AffOrgMatchVoter}s for {@link #createDocOrgRelationMatcher(JavaSparkContext, String, String, String, Float)} 
+     */
+    public static ImmutableList<AffOrgMatchVoter> createDocOrgRelationMatcherVoters() {
+        return of(
```
Comment (mpol) [inline iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffOrgMatcherFactory.java]: It seems more readable to use it as `ImmutableList.of(...)` like they do in the examples in the documentation.

File: iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchingResultPrinter.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+ * A helper that prints results of affiliation matching (actual matched affiliations in relation to
+ * the expected ones). Used in classes that test affiliation matching quality. 
+ * 
+ *  
+ * @author Łukasz Dumiszewski
+*/
+
+public class AffMatchingResultPrinter {
+    
+
```
Comment (mpol) [inline iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchingResultPrinter.java]: Section label?

File: iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffOrgMatcherFactory.java (hunk 1 above)
Comment (lukdumi) [inline iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffOrgMatcherFactory.java]: Well, maybe... OK, I'll change it.

File: iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchingResultPrinter.java (hunk 2 above)
Comment (lukdumi) [inline iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchingResultPrinter.java]: :+1:

File: iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/match/voter/AffOrgMatchVoterStrengthEstimator.java [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+            
+            
+            // log
+            
+            calcAndPrintResult(of("src/test/resources/experimentalData/expectedOutput/set1/matched_aff.json",
+                                  "src/test/resources/experimentalData/expectedOutput/set2/matched_aff.json",
+                                  "src/test/resources/experimentalData/expectedOutput/set4/matched_aff.json"));
+            
+            
+            FileUtils.deleteDirectory(new File(outputDirPath));
```
Comment (mpol) [inline iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/match/voter/AffOrgMatchVoterStrengthEstimator.java]: `finally`?

Comment (lukdumi) [inline iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/match/voter/AffOrgMatchVoterStrengthEstimator.java]: ?

**Pull request closed without merging**